### PR TITLE
Create SuccessfulTransactionMessage component

### PR DESCRIPTION
**Pull request by therealemjy**
- add `check` icon to `Icon` component
- update list of component imports at `src/components/index.ts`
- extract `BscLink` components from `AuthModal`
- create `SuccessfulTransactionMessage` component
- create `generateBscScanAddressUrl` util to generate a URL on BSC scan from an address

Screenshot 2022-04-05 at 09 49 36

### CODE REVIEW

File: src/components/v2/BscLink/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+export interface IBscLinkProps {
+  hash: string;
+  className?: string;
+}
+
+export const BscLink: React.FC<IBscLinkProps> = ({ hash, className }) => {
+  const styles = useStyles();
+
+  const bscScanUrl = `https://${isOnTestnet ? 'testnet.' : ''}bscscan.com/address/${hash}`;
```
Comment (baomastr) [inline src/components/v2/BscLink/index.tsx]: can be a good util that wraps urls. wouldn't it used in other places?

**Review by baomastr — APPROVED** [review]

File: src/components/v2/BscLink/index.tsx (hunk 1 above)
Comment (therealemjy) [reply src/components/v2/BscLink/index.tsx]: Yeah good point. It could be reused in the footer already. I'll make the change 👍

Comment (therealemjy) [reply src/components/v2/BscLink/index.tsx]: Done

File: src/components/v2/Layout/Footer/index.tsx [hunk 2]
```diff
@@ -37,7 +38,12 @@ export const Footer: React.FC<IFooterProps> = ({ currentBlockNumber }) => {
       </Typography>
 
       <div css={styles.links}>
-        <a css={styles.link} href={ETHERSCAN_XVS_URL} target="_blank" rel="noreferrer">
+        <a
```
Comment (coreyar) [inline src/components/v2/Layout/Footer/index.tsx:41]: Should we use typography here?

**Review by coreyar — APPROVED** [review]

File: src/components/v2/Layout/Footer/index.tsx (hunk 2 above)
Comment (therealemjy) [reply src/components/v2/Layout/Footer/index.tsx:41]: We shouldn't no, the anchor tag is used as a wrapper only here, we don't display any text inside it.

**Pull request merged**